Scripts/AwsEC2GetPublicSGRules/AwsEC2GetPublicSGRules.yml

```diff
@@ -0,0 +1,92 @@
+args:
+- default: false
+  description: Security Group ID (sg-xxxxxxxxx)
+  isArray: false
+  name: groupId
+  required: true
+  secret: false
+- default: false
+  description: JSON string of the ipPermissions
```

Comment:
give an example of the JSON

Reply:
Added more description and link to the sample